component/booksCard.jsx

```diff
@@ -29,6 +33,25 @@ export default function BooksCard({ apikey }) {
                 <Link href={`/books/${book.primary_isbn13}`}>
                   <a>More details → </a>
                 </Link>
+
+                {isinwishlist ? (
+                  <button
+                    onClick={() => {
+                      // remove from wishlist
```

Comment:
Just my personal preference: In general comments are good but I think you could remove this one because it's totally clear what `deleteFromWishList` does.

Reply:
fix